src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -24,6 +24,7 @@ class FlattenException
 {
     private $message;
     private $code;
+    private $title;
```

Comment:
having a title is another feature not required by this PR, I'd prefer not adding it here but well, in another PR.